backend/app/tests/crud/test_llm.py

```
@@ -1,49 +1,42 @@
 
 @pytest.fixture
 def test_project(db: Session) -> Project:
-    """Get the first available test project."""
-    project = db.exec(select(Project).limit(1)).first()
-    assert project is not None, "No test project found in seed data"
-    return project
+    return get_project(db)
 
 
 @pytest.fixture
 def test_organization(db: Session, test_project: Project) -> Organization:
```

Ruff (0.15.7)

[warning] Unused function argument: `test_project`

(ARG001)